sdk/core/src/SelfBackendVerifier.ts

```
@@ -1,285 +1,291 @@
-        typeof error === 'object' &&
-        error.message &&
-        error.message.includes('INVALID_FORBIDDEN_COUNTRIES')
-      ) {
-        errorMessage =
-          'The forbidden countries list in the backend does not match the list provided in the frontend SDK. Please ensure both lists are identical.';
-      }
+      verificationConfig = await this.configStorage.getConfig(configId);
+    } catch (error) {
+      issues.push({ type: ConfigMismatch.ConfigNotFound, message: 'Config not found' });
+    } finally {
+      if (!verificationConfig) throw new ConfigMismatchError(issues);
```

Biome (1.9.4)

[error] Unsafe usage of 'throw'.

'throw' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)